apps/studio-next/src/app/page.tsx

```diff
@@ -0,0 +1,115 @@
+import Image from 'next/image'
+
+export default function Home() {
+  return (
+    <main className="flex min-h-screen flex-col items-center justify-between p-24">
```

Comment:
what about clearing the template and only have a text saying `Home`? we are going to remove these anyway.